packages/webawesome/src/components/rating/rating.css

```
@@ -4,7 +4,7 @@
   --size-m: var(--wa-space-l);
   --size-l: var(--wa-space-xl);
 
-  --symbol-color: var(--wa-color-neutral-fill-normal);
+  --symbol-color: var(--wa-color-neutral-border-loud);
```

```suggestion
  --symbol-color: var(--wa-color-neutral-on-quiet);
```

This will be a safer bet to ensure compliant contrast.